domains/node.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Personal portfolio, I know SSL won't work :/",
+  "repo": "",
+  "owner": {
+    "username": "node0505",
+    "email": "",
+    "discord": "node#0505"
+  },
+  "record": {
+    "CNAME": "website-gold-psi.vercel.app"
```

Comment:
Vercel will not work with a CNAME, you could use [Railway](https://railway.app?referralCode=jj), or use a `URL` record